CONTRIBUTING.md

```diff
@@ -74,10 +74,28 @@ $ composer test
 > **Note:** The `composer test` command is a wrapper around `phpunit`, so you can use all the common [options](https://phpunit.de/manual/5.7/en/textui.html#textui.clioptions) that you would with `phpunit`. However you need prepend the options list with the additional `--` dashes that `composer` requires:
 
 ```bash
-# Run a suite and a filter
-$ composer test -- --testsuite=unit --filter=Predis
+# Run only unit tests
+$ composer test -- --testsuite=unit
 ```
 
+```bash
+# Run only integration tests
+$ composer test -- --testsuite=integration
+```
+
+Testing integrations with libraries requires an additional step, as there are different scenarios where you want to test
+a specific integration. You can find available scenarios in `composer.json` at key `extras.scenarios`.
+
+As an example, in order to run Guzzle tests with Guzzle v5 library, run:
+
+    # Only the first time, to create all the different scenarios
+    $ composer scenario:update
```

Comment:
After running this command, the `LICENSE` file gets updated. Here's the diff:

```diff
diff --git a/LICENSE b/LICENSE
index be5bf07..1e187aa 100644
--- a/LICENSE
+++ b/LICENSE
@@ -27,3 +27,16 @@ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
 CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
 OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+DEPENDENCY LICENSES:
+
+Name                       Version      License
+ircmaxell/password-compat  v1.0.4       MIT
+opentracing/opentracing    1.0.0-beta5  MIT
+paragonie/random_compat    v9.99.99     MIT
+psr/http-message           1.0.1        MIT
+psr/log                    1.1.0        MIT
+symfony/polyfill-php55     v1.10.0      MIT
+symfony/polyfill-php56     v1.10.0      MIT
+symfony/polyfill-php70     v1.10.0      MIT
+symfony/polyfill-util      v1.10.0      MIT
```

Reply:
Thanks for noticing it, let me look at it!